tests/tests/swan_tests.cpp

```diff
@@ -114,6 +114,10 @@ struct swan_fixture : database_fixture {
       generate_blocks( HARDFORK_CORE_216_TIME );
       generate_block();
     }
+    void wait_for_hf_core_1270() {
+       generate_blocks( HARDFORK_CORE_1270_TIME );
+       generate_block();
+    }
```

Comment:
To be more accurate, here we should generate blocks until the maintenance time but not the hf time.